README.md

```diff
@@ -3,82 +3,21 @@
-In some cases you may want to experiment with running the latest development version. You can install this 
-version with the following command:
-```
-pip install git+https://github.com/fgrunewald/polyply_1.0.git#polyply_1.0
-```
-The behavior of the pip command can vary depending of the specificity of your python installation. See the 
-[documentation on installing a python package][pipdoc] to learn more.
-
-### Polymer Library
-Some macromolecules are implemented in our library for a range of different force-fields.
-To get a list of all libraries available run:
-```
-polyply -list-lib
-```
-To get a detailed list of all the individual macromolecules:
-```
-polyply -list-blocks <Force Field Name>
-```
-Note that while you can combine fragments from different libraries (e.g. the martini-3 polymer
-library with martini-3 proteins), we cannot guarantee that all the links are present to make a
-meaningful itp. All blocks within a library can safely be combined. For more information on how
-to implement links between different blocks see the [wiki].
-
-### Itp file generation
-To generate a linear polymer chain using parameters, provided in the library run:
-```
-polyply gen_itp -lib <library_name> -name <name> -seq <monomer:#number> -o <name_outfile + .itp>
-```
-
-For more information on how to generate itp files for more complex polymers or how
-to combine them with existing itp files see the [wiki].
-
-### Initial structure generation
-To generate an initial structure run:
-```
-polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule> -dens <density>
-```
-or:
-```
-polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule> -box <x> <y> <z>
-```
-In order to append coordinates to an already existing coordinate file run:
-```
-polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule> -c <init_coords.gro> -box <x> <y> <z>
-```
-or
-```
-polyply gen_coords -p <top> -o <name_outfile + .gro> -name <name of molecule> -c <init_coords.gro> -dens <density>
-```
-Note that at the moment polyply can only generate disordered structures of polymers. All molecules
-that have secondary structure (e.g. DNA, proteins) cannot be generated accurately. At the moment
-chirality is also not taken into account. All polymers are atactic unless a dihedral specifies the 
-chirality.
+## Quick references
+[Installation Guide](https://github.com/marrink-lab/polyply_1.0/wiki/Installation)<br/>
+[FAQs](https://github.com/marrink-lab/polyply_1.0/wiki/FAQs)<br/>
+[Submissions to martini polymer library](https://github.com/marrink-lab/polyply_1.0/wiki/Submit-polymer-parameters)<br/>
+[Tutorial Martini Polymers](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-martini-polymer-melts)<br/>
+[Tutorial GROMOS Polymers](https://github.com/marrink-lab/polyply_1.0/wiki/Tutorial:-GROMOS-polymer-melts)<br/>
```

Comment:
The `<br\>` line breaks look weird in markdown. Change them to `\` (https://stackoverflow.com/questions/26626256/how-to-insert-a-line-break-br-in-markdown)

Reply:
I like the quick references. 

Small thingy: you are capitalizing "Installation Guide" and "Tutorial Martini Polymers" but not "Submissions to martini polymer library" - I would be consistent.